packages/zoe/test/swingsetTests/zoe-metering/test-zoe-metering.js

```diff
@@ -15,7 +15,7 @@ const CONTRACT_FILES = [
 ];
 const generateBundlesP = Promise.all(
   CONTRACT_FILES.map(async contract => {
-    const bundle = await bundleSource(`${__dirname}/${contract}`);
+    const bundle = { bundle: await bundleSource(`${__dirname}/${contract}`) };
```

Comment:
see previous about removing the extra `.bundle` layer

Reply:
done.